src/sandstorm/grain.capnp

```diff
@@ -528,7 +528,12 @@ struct SturdyRefOwner {
   # Human-readable explanation of how this capability relates to the grain. This is displayed
   # when visualizing outgoing capabilities from the calling grain in the Sandstorm UI. This string
   # will NOT be communicated to the host of the saved capability; it is intercepted and used by the
-  # Sansdtorm system.
+  # Sandstorm system.
+
+  userId @1 :Data;
+  # ID of the user whose permissions are tied to this capability. This data will NOT be communicated
```

Comment:
I don't quite understand what's going on here. Are you sure this belongs in `SturdyRefOwner`, which is meant to identify the owner of the reference itself (i.e., the grain), not the owner of the referenced object? Normally the grain fills in `SturdyRefOwner` when making an outgoing `save()` request on some arbitrary capability it has somehow received.